### PR TITLE
Send psmgr-style '{heartbeat: true}' to handle_event

### PR DESCRIPTION
**Pull request by stillinbeta**

### CODE REVIEW

File: src/chunked_parser.erl [hunk 1]
```diff
@@ -38,6 +38,9 @@ parse_msgs(Data, Callback, CbState) ->
                     parse_msgs(Rest1, Callback, CbState);
                 {ok, Chunk, Rest1} ->
                     case (catch mochijson2:decode(Chunk)) of
+                        {struct, [{<<"heartbeat">>, true}]} ->
```
Comment (omarkj) [inline src/chunked_parser.erl]: Does psmgr send `{"hearbeat":true}` or `{"hearbeat":"true"}`? They get decoded differently:

```
mochijson2:decode(<<"{\"heartbeat\": \"true\"}">>).
{struct,[{<<"heartbeat">>,<<"true">>}]}
mochijson2:decode(<<"{\"heartbeat\": true}">>).
{struct,[{<<"heartbeat">>,true}]}
```

Comment (stillinbeta) [inline src/chunked_parser.erl]:
```
20:45:14.727728 <0.10223.0> procs_callback:handle_msg([{<<"heartbeat">>,true}], {cb_state,"0-88923082:1-67142462:2-65377872",<<"psmgr.26813">>,true,0})
```

Comment (omarkj) [inline src/chunked_parser.erl]: Discussed in real life: tracing shows it as atoms so this is correct.

Comment (stillinbeta) [inline src/chunked_parser.erl]: that's the trace

**Comment by omarkj** [comment]
un plus

**Comment by evanmcc** [comment]
+1

**Pull request merged**